Clarifications to creating-projects-using-cli (#862)
o3de/o3de.org: content/docs/welcome-guide/create/creating-projects-using-cli.md

        ```

        {{< important >}}

> If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor _and_ other tools from the installed engine's build directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.

Reviewer: Possible reword for clarity - If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor _and_ other tools from the directory where the engine is installed, _not_ the directory where 
 the engine was built from. The Windows install directory typically ends in `/bin/Windows/profile`.

Reviewer 2: This reword introduces a passive voice, which we try to avoid in these docs. I'm inclined to prefer the current proposed change.

        {{< /important >}}

You can also run Project Manager (`o3de.exe`) from the same directory to edit your project's settings, add or remove Gems from the project, rebuild your project, and launch the Editor.

{{< caution >}}

> When you launch the Editor, the **Asset Processor** from the same directory will also launch.  You must close all **Asset Processor** tasks before you attempt to launch the Editor from a different directory.

Reviewer: Reword suggestion for clarity - When you launch the Editor, the **Asset Processor** that resides in that directory will be launched. Any existing **Asset Processor** tasks that are running must be closed before you attempt to launch the Editor from a different directory.

Reviewer 2: I agree that this note could be improved, but this rewording still feels a bit awkward to me.

Reviewer 2: Starting the second sentence with an infinitive phrase might help. For example, "To launch the Editor from a different directory, you must close any **Asset Processor** tasks that are running."

Author: > To launch the Editor from a different directory, you must close any **Asset Processor** tasks that are running.

I agree, this is a better rewording.  I'll make the change pending any other comments.

Reviewer: Yeah that sounds good

> If you launch the Editor from the wrong directory, the **Asset Processor** processor from that directory will also launch.  You must close all **Asset Processor** tasks before you attempt to launch the Editor from a different directory.

Reviewer 2: I think that use of the word "wrong" here might not be well-defined or even accurate in every case where this cautionary note is relevant. Instead, I think it's good as a general purpose note. For example, it's important to know when intentionally switching from one project to another.
```suggestion
When you launch the Editor, the **Asset Processor** from the same directory will also launch.  You must close all **Asset Processor** tasks before you attempt to launch the Editor from a different directory.
```
